### ISSUE TITLE
Woocommerce Booking Product Shows 0 Price & Products With Display Price show 0 price

### ISSUE THREAD

**Issue by actionculture** [facebookincubator/facebook-for-woocommerce#106]
I have the latest version of the FB for WC plugin and until the latest update all of my products synced well, with prices but now everything syncs but without prices. Please help.

Most of my products have a Base cost set, not a Display cost. All are Bookable Products.

Thanks in advance,
Glenn

**Comment by mengyingdu** [comment]
Duplicate of #50

**Comment by mengyingdu** [comment]
Hi actionculture,

Have you tried [v1.6.4](https://github.com/facebookincubator/facebook-for-woocommerce/releases)?
It should support bookable products with 'Display Cost' set.

If the display cost is not set in general whatever get_display_cost returns by the WC_Bookings plugin is used. (This appears to be base_cost + block_cost in the average case.)
Please try the new version and force resync.

If you still have this issue with new version after resync, let us know.

**Comment by cabgfx** [comment]
Hi, I have a similar issue, with the [WooCommerce Composite Products](https://woocommerce.com/products/composite-products/) extension.

The add-on options for our products are used to calculate the price, thus the “base” product isn't priced. Instead, prices are displayed as _From $50_ (handled entirely by the plugin).

Using plugin 1.6.4 + WP 4.8.2 + Woo 3.2.1

What further details can I provide, that may help? Thanks!

**Comment by dmitridr** [comment]
@cabgfx 

I recommend opening a new issue with us for the specific extension you mention and our support of it. 
As for the details:

- Are these products syncing with 0 price or not syncing? (I thought I fixed the 0 price in 1.6.4, if you Force Resync these should be marked invisible and if you reset these should not sync at all.)
- [This](https://github.com/facebookincubator/facebook-for-woocommerce/blob/master/includes/fbproduct.php#L92) is how we solved the issue for bookable products, if there is a similar solution for Composite products, we'd be happy to review your pull request to support them.
